### DISCUSSION TITLE
not found: function: 'T::EQUAL'

### DISCUSSION THREAD

**Discussion by ngirard** [MontFerret/ferret#662] [Q&A] [answered]
Using latest precompiled binary on Ubuntu 20.04.
Using `T::EQUAL` in a query results in this error:
```
Failed to compile the query
not found: function: 'T::EQUAL'
```

This is a blocking problem for me.

**Comment by ziflex** [comment, marked as answer]
Hey,

Sorry for the inconvenience, this is a mistake in docs. It should be ``T::EQ``.

**Comment by ngirard** [comment]
Indeed, thanks for your quick reply!

I'm willing to amend the docs, but it's unlikely to happen before next week-end.
